website/docs/r/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: - "Cloud"
+layout: "aci"
+page_title: "ACI: aci_cloud_service_epg"
+sidebar_current: "docs-aci-resource-cloud_service_epg"
+description: |-
+  Manages ACI Cloud Service EPG
+---
+
+# aci_cloud_service_epg #
+
+Manages ACI Cloud Service EPG
+Note: This resource is supported in Cloud APIC only.
+
+## API Information ##
+
+* `Class` - cloudSvcEPg
+* `Distinguished Name` - uni/tn-{tenant_name}/cloudapp-{application_name}/cloudsvcepg-{name}
+
+## GUI Information ##
+
+* `Location` - Application Management -> EPGs
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_cloud_service_epg" "example" {
+  cloud_application_container_dn  = aci_cloud_applicationcontainer.example.id
+  name                            = "example"
+  access_type                     = "Public"
+  deployment_type                 = "CloudNative"
+  flood_on_encap                  = "disabled"
+  label_match_criteria            = "AtleastOne"
+  cloud_service_epg_type          = "Azure-SqlServer"
+  cloud_rs_cloud_epg_ctx          = aci_vrf.example.id
+}
+```
+
+## Argument Reference ##
+
+* `cloud_application_container_dn` - (Required) Distinguished name of the parent Cloud Application container object. Type: String.
+* `name` - (Required) Name of the Cloud Service EPG object. Type: String.
+* `annotation` - (Optional) Annotation of the Cloud Service EPG object. Type: String.
+* `name_alias` - (Optional) Name Alias of the Cloud Service EPG object. Type: String.
+* `access_type` - (Optional) This refers to the type of connectivity to the service. It could be a public or private connectivity. Allowed values are "Private", "Public", "PublicAndPrivate", "Unknown", and default value is "Public". Type: String.
+* `azure_private_endpoint` - (Optional) Naming for Azure Private Endpoint created from the Service Cloud EPG. Naming override for any Azure Private Endpoint that gets created from this service EPG. The set of variable supported by the naming override is the same of those supported in the global naming policy. However, there is no mandatory variable enforced by validations. Type: String.
```

Comment:
-> Naming for Azure Private Endpoint created from the Service Cloud EPG. The set of variables supported by the naming override is the same as those supported in the global naming policy. However, there is no mandatory variable enforced by validations. Type: String.